Diary/Diary/WeeklyTask.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Diary
+{
+    abstract public class WeeklyTask
+    {
+        internal string name;
+        internal abstract string GetAlarm();
+        internal WeeklyTask(string name)
+        {
+        }        
+    }
```

Comment:
fields must be private
empty lines
reorder methods

```suggestion
        private string name;
      
        internal WeeklyTask(string name)
        {
        }        
        
        internal abstract string GetAlarm();
    }
```